index.html

```diff
@@ -168,9 +168,9 @@ <h2 class="cv-cell-title">Skills</h2>
             </div>           
           </section>
           <section class="flex form-container">
-            <p class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat 
-              please get in touch.</p>            
-            <form class="form flex" action="https://formspree.io/f/{form_id}" method="post">
+            <h1 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat 
+              please get in touch.</h1>            
```

Comment:
- [ ] (OPTIONAL) Kindly make this a `h2` tag or an `p` tag for best semantic practices as your site should have only one main `h1` which should be the main heading 👍